### ISSUE TITLE
Documentation is outdated for client

### ISSUE THREAD

**Issue by stuartmac** [bartongroup/slivka#126]
The client docs linked to from the compbio slivka [here](http://bartongroup.github.io/slivka/python_client.html#python) seem outdated. In particular, there is no `Service.form` attribute or `Service.new_form()` method. Looks like this may have been replaced with `Service.parameters`?

**Comment by warownia1** [comment]
Hi, how did you land on that page? The version number in the left corner says that the page is for version 0.7 while the rest of the documentation is for 0.8.1. It's a page from the old build still lingering around. The link pointing to that page needs fixing.

**Comment by stuartmac** [comment]
Yeah I noticed the version inconsistency too. The link was from "client documentation" on the Getting Started page: https://www.compbio.dundee.ac.uk/slivka/gettingstarted.

**Comment by warownia1** [comment]
The client documentation moved to https://github.com/bartongroup/slivka-python-client README file.
